src/components/context/TodoContext.tsx

```diff
@@ -0,0 +1,50 @@
+import { createContext, useContext, useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { getTodos } from '../../api';
+
+type FilterType = 'All' | 'Active' | 'Completed';
```

Comment:
To wyniósłbym do jakiegos innego pliku. Mozesz tez pomyslec o tym aby te wartosci trzymac w Enumie. Mówie o tym dlatego ze np twój komponent TodoFilter tez korzysta z tych wartosci